e2e/Makefile

```diff
@@ -1,39 +1,57 @@
-KUBERNETES_VERSION = 1.23.4
+# NOTE: kind version is specified at aqua.yaml
+E2ETEST_K8S_VERSION  := 1.24.0
+CERT_MANAGER_VERSION := 1.8.2
 
-KIND_CONFIG = kind-config.yaml
+PROJECT_DIR := $(CURDIR)/../
+BIN_DIR := $(PROJECT_DIR)/bin
+
+CURL := curl -sSLf
+KIND := $(BIN_DIR)/kind
```

Comment:
This definition is not used because it is installed with aqua.

Reply:
That's true. Thank you. I remove it.